apps/evm/src/pages/Market/CorePoolMarket/__tests__/__snapshots__/indexApyCharts.spec.tsx.snap

```diff
@@ -1,8 +1,8 @@
 // Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
 
-exports[`CorePoolMarket - Feature flag enabled: apyCharts > fetches market details and displays them correctly 1`] = `"Supply info0%Total supplied> $100T / > $100T> 100T / > 100T XVSCurrent APY0.16%"`;
+exports[`CorePoolMarket - Feature flag enabled: apyCharts > fetches market details and displays them correctly 1`] = `"Supply info30D6M1Y0%Total supplied> $100T / > $100T> 100T / > 100T XVSAverage APY< 0.01%Current APY0.16%Supply APY"`;
 
-exports[`CorePoolMarket - Feature flag enabled: apyCharts > fetches market details and displays them correctly 2`] = `"Borrow info< 0.01%Total borrowed$2.36M / > $100T1.85M / > 100T XVSCurrent APY-6.48%Liquidation Threshold50%Liquidation Penalty10%"`;
+exports[`CorePoolMarket - Feature flag enabled: apyCharts > fetches market details and displays them correctly 2`] = `"Borrow info30D6M1Y< 0.01%Total borrowed$2.36M / > $100T1.85M / > 100T XVSAverage APY2.01%Current APY-6.48%Liquidation Threshold50%Liquidation Penalty10%Borrow APY"`;
```

Comment:
These changes are making the snapshots closer to reality by showing the data fetched from queries. I believe the previous iteration was stuck on the loading state, thus showing incomplete data.